app/views/chunks/get-help-with-claim.njk

```diff
@@ -0,0 +1,7 @@
+{% block content %}
+  <div>
+    <h2 class="govuk-heading-m">Get help with your claim</h2>
+    <p class="govuk-body">If you need help with your claim, contact the Rural Payments Agency and give your agreement number.</p>
+    {{ rpaContactDetails({email: ruralPaymentsAgency.email, callChargesUri: ruralPaymentsAgency.callChargesUri, telephone: ruralPaymentsAgency.telephone}) }}
+  </div>
+{% endblock %}
```

Comment:
We have a folder called macros which we could use instead of creating one called chunks

Reply:
Also are there other places in the app where we could use this reusable section?

Reply:
Macros are for njk macros , this  are no macros , it has no functionality just just content , that't the reason is apart, there it is better to keep it a part as i see many other "chunks" created to re-use code njks